docs/reference/mapping/types/dense-vector.asciidoc

```diff
@@ -9,7 +9,8 @@ not exceed 500. The number of dimensions can be
 different across documents. A `dense_vector` field is
 a single-valued field.
 
-These vectors can be used for document scoring.
+These vectors can be used for
+{ref}/query-dsl-script-score-query.html#vector-functions[document scoring].
```

Comment:
is there a reason not use an internal link, eg. `<<vector-functions,document scoring>>`?

Reply:
I'm still a bit confused, this is the same document, isn't it?

Reply:
@jpountz  Sorry Adrien, I meant that inside one asciidoc doc `dense-vector.asciidoc` we want to reference a section of another  asciidoc doc `script-score-query.asciidoc`. 

We can indeed use an easier format : <<query-dsl-script-score-query,`document_scoring`>>, but this will link to the whole document.  And as I understood after talking with the documentation team, the only way to link to the **section** of another doc is to use this full html link.

Reply:
@jpountz  Sorry Adrien, please disregard my previous comments. I have followed your advice to use internal links and it looks like documentation CI passed.